Algorithms/Strings/NaiveStringSearch.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+
+/// Implements the traditional naive string matching algorithm in C# for TheAlgorithms/C-Sharp.
+namespace Algorithms.Strings
+{
+   public static class NaiveStringSearch
+   {
+	 		/// <summary>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=4766632)